packages/web/docs/src/components/call-to-action.tsx

```diff
@@ -20,6 +20,10 @@ const variantStyles = {
     'bg-green-800 hover:bg-green-700 text-white' +
       ' focus-visible:outline-2 focus-visible:outline-offset-1 focus-visible:outline-green-800/40',
   ),
+  tertiary: cn(
+    'bg-transparent text-green-1000' +
+      ' focus-visible:outline-2 focus-visible:outline-offset-1 focus-visible:outline-green-800',
```

Comment:
```suggestion
      'focus-visible:outline-2 focus-visible:outline-offset-1 focus-visible:outline-green-800',
```

Reply:
This space is needed there because I use `+`.

I probably should just surrender and let VSCode wrap lines, right?

Reply:
> I probably should just surrender and let VSCode wrap lines, right?

yeah 🥲